generated/nirfmxbluetooth/nirfmxbluetooth.proto

```diff
@@ -1,6 +1,6 @@
 
 //---------------------------------------------------------------------
-// This file is generated from NI-RFMXBLUETOOTH API metadata version 24.3.0
+// This file is generated from NI-RFMXBLUETOOTH API metadata version 24.5.0
```

Comment:
Make sure to update [the README](https://github.com/ni/grpc-device/blob/main/README.md#supported-ni-drivers) as well for all these version bumps

Reply:
Updated the Readme.MD file